<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>An√°lisis metilaci√≥n O. maya</title><style>
/* cspell:disable-file */
/* webkit printing magic: print all background colors */
html {
	-webkit-print-color-adjust: exact;
}
* {
	box-sizing: border-box;
	-webkit-print-color-adjust: exact;
}

html,
body {
	margin: 0;
	padding: 0;
}
@media only screen {
	body {
		margin: 2em auto;
		max-width: 900px;
		color: rgb(55, 53, 47);
	}
}

body {
	line-height: 1.5;
	white-space: pre-wrap;
}

a,
a.visited {
	color: inherit;
	text-decoration: underline;
}

.pdf-relative-link-path {
	font-size: 80%;
	color: #444;
}

h1,
h2,
h3 {
	letter-spacing: -0.01em;
	line-height: 1.2;
	font-weight: 600;
	margin-bottom: 0;
}

.page-title {
	font-size: 2.5rem;
	font-weight: 700;
	margin-top: 0;
	margin-bottom: 0.75em;
}

h1 {
	font-size: 1.875rem;
	margin-top: 1.875rem;
}

h2 {
	font-size: 1.5rem;
	margin-top: 1.5rem;
}

h3 {
	font-size: 1.25rem;
	margin-top: 1.25rem;
}

.source {
	border: 1px solid #ddd;
	border-radius: 3px;
	padding: 1.5em;
	word-break: break-all;
}

.callout {
	border-radius: 3px;
	padding: 1rem;
}

figure {
	margin: 1.25em 0;
	page-break-inside: avoid;
}

figcaption {
	opacity: 0.5;
	font-size: 85%;
	margin-top: 0.5em;
}

mark {
	background-color: transparent;
}

.indented {
	padding-left: 1.5em;
}

hr {
	background: transparent;
	display: block;
	width: 100%;
	height: 1px;
	visibility: visible;
	border: none;
	border-bottom: 1px solid rgba(55, 53, 47, 0.09);
}

img {
	max-width: 100%;
}

@media only print {
	img {
		max-height: 100vh;
		object-fit: contain;
	}
}

@page {
	margin: 1in;
}

.collection-content {
	font-size: 0.875rem;
}

.column-list {
	display: flex;
	justify-content: space-between;
}

.column {
	padding: 0 1em;
}

.column:first-child {
	padding-left: 0;
}

.column:last-child {
	padding-right: 0;
}

.table_of_contents-item {
	display: block;
	font-size: 0.875rem;
	line-height: 1.3;
	padding: 0.125rem;
}

.table_of_contents-indent-1 {
	margin-left: 1.5rem;
}

.table_of_contents-indent-2 {
	margin-left: 3rem;
}

.table_of_contents-indent-3 {
	margin-left: 4.5rem;
}

.table_of_contents-link {
	text-decoration: none;
	opacity: 0.7;
	border-bottom: 1px solid rgba(55, 53, 47, 0.18);
}

table,
th,
td {
	border: 1px solid rgba(55, 53, 47, 0.09);
	border-collapse: collapse;
}

table {
	border-left: none;
	border-right: none;
}

th,
td {
	font-weight: normal;
	padding: 0.25em 0.5em;
	line-height: 1.5;
	min-height: 1.5em;
	text-align: left;
}

th {
	color: rgba(55, 53, 47, 0.6);
}

ol,
ul {
	margin: 0;
	margin-block-start: 0.6em;
	margin-block-end: 0.6em;
}

li > ol:first-child,
li > ul:first-child {
	margin-block-start: 0.6em;
}

ul > li {
	list-style: disc;
}

ul.to-do-list {
	padding-inline-start: 0;
}

ul.to-do-list > li {
	list-style: none;
}

.to-do-children-checked {
	text-decoration: line-through;
	opacity: 0.375;
}

ul.toggle > li {
	list-style: none;
}

ul {
	padding-inline-start: 1.7em;
}

ul > li {
	padding-left: 0.1em;
}

ol {
	padding-inline-start: 1.6em;
}

ol > li {
	padding-left: 0.2em;
}

.mono ol {
	padding-inline-start: 2em;
}

.mono ol > li {
	text-indent: -0.4em;
}

.toggle {
	padding-inline-start: 0em;
	list-style-type: none;
}

/* Indent toggle children */
.toggle > li > details {
	padding-left: 1.7em;
}

.toggle > li > details > summary {
	margin-left: -1.1em;
}

.selected-value {
	display: inline-block;
	padding: 0 0.5em;
	background: rgba(206, 205, 202, 0.5);
	border-radius: 3px;
	margin-right: 0.5em;
	margin-top: 0.3em;
	margin-bottom: 0.3em;
	white-space: nowrap;
}

.collection-title {
	display: inline-block;
	margin-right: 1em;
}

.page-description {
    margin-bottom: 2em;
}

.simple-table {
	margin-top: 1em;
	font-size: 0.875rem;
	empty-cells: show;
}
.simple-table td {
	height: 29px;
	min-width: 120px;
}

.simple-table th {
	height: 29px;
	min-width: 120px;
}

.simple-table-header-color {
	background: rgb(247, 246, 243);
	color: black;
}
.simple-table-header {
	font-weight: 500;
}

time {
	opacity: 0.5;
}

.icon {
	display: inline-block;
	max-width: 1.2em;
	max-height: 1.2em;
	text-decoration: none;
	vertical-align: text-bottom;
	margin-right: 0.5em;
}

img.icon {
	border-radius: 3px;
}

.user-icon {
	width: 1.5em;
	height: 1.5em;
	border-radius: 100%;
	margin-right: 0.5rem;
}

.user-icon-inner {
	font-size: 0.8em;
}

.text-icon {
	border: 1px solid #000;
	text-align: center;
}

.page-cover-image {
	display: block;
	object-fit: cover;
	width: 100%;
	max-height: 30vh;
}

.page-header-icon {
	font-size: 3rem;
	margin-bottom: 1rem;
}

.page-header-icon-with-cover {
	margin-top: -0.72em;
	margin-left: 0.07em;
}

.page-header-icon img {
	border-radius: 3px;
}

.link-to-page {
	margin: 1em 0;
	padding: 0;
	border: none;
	font-weight: 500;
}

p > .user {
	opacity: 0.5;
}

td > .user,
td > time {
	white-space: nowrap;
}

input[type="checkbox"] {
	transform: scale(1.5);
	margin-right: 0.6em;
	vertical-align: middle;
}

p {
	margin-top: 0.5em;
	margin-bottom: 0.5em;
}

.image {
	border: none;
	margin: 1.5em 0;
	padding: 0;
	border-radius: 0;
	text-align: center;
}

.code,
code {
	background: rgba(135, 131, 120, 0.15);
	border-radius: 3px;
	padding: 0.2em 0.4em;
	border-radius: 3px;
	font-size: 85%;
	tab-size: 2;
}

code {
	color: #eb5757;
}

.code {
	padding: 1.5em 1em;
}

.code-wrap {
	white-space: pre-wrap;
	word-break: break-all;
}

.code > code {
	background: none;
	padding: 0;
	font-size: 100%;
	color: inherit;
}

blockquote {
	font-size: 1.25em;
	margin: 1em 0;
	padding-left: 1em;
	border-left: 3px solid rgb(55, 53, 47);
}

.bookmark {
	text-decoration: none;
	max-height: 8em;
	padding: 0;
	display: flex;
	width: 100%;
	align-items: stretch;
}

.bookmark-title {
	font-size: 0.85em;
	overflow: hidden;
	text-overflow: ellipsis;
	height: 1.75em;
	white-space: nowrap;
}

.bookmark-text {
	display: flex;
	flex-direction: column;
}

.bookmark-info {
	flex: 4 1 180px;
	padding: 12px 14px 14px;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
}

.bookmark-image {
	width: 33%;
	flex: 1 1 180px;
	display: block;
	position: relative;
	object-fit: cover;
	border-radius: 1px;
}

.bookmark-description {
	color: rgba(55, 53, 47, 0.6);
	font-size: 0.75em;
	overflow: hidden;
	max-height: 4.5em;
	word-break: break-word;
}

.bookmark-href {
	font-size: 0.75em;
	margin-top: 0.25em;
}

.sans { font-family: ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol"; }
.code { font-family: "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace; }
.serif { font-family: Lyon-Text, Georgia, ui-serif, serif; }
.mono { font-family: iawriter-mono, Nitti, Menlo, Courier, monospace; }
.pdf .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK JP'; }
.pdf:lang(zh-CN) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK SC'; }
.pdf:lang(zh-TW) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK TC'; }
.pdf:lang(ko-KR) .sans { font-family: Inter, ui-sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, "Apple Color Emoji", Arial, sans-serif, "Segoe UI Emoji", "Segoe UI Symbol", 'Twemoji', 'Noto Color Emoji', 'Noto Sans CJK KR'; }
.pdf .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .code { font-family: Source Code Pro, "SFMono-Regular", Menlo, Consolas, "PT Mono", "Liberation Mono", Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.pdf .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK JP'; }
.pdf:lang(zh-CN) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK SC'; }
.pdf:lang(zh-TW) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK TC'; }
.pdf:lang(ko-KR) .serif { font-family: PT Serif, Lyon-Text, Georgia, ui-serif, serif, 'Twemoji', 'Noto Color Emoji', 'Noto Serif CJK KR'; }
.pdf .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK JP'; }
.pdf:lang(zh-CN) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK SC'; }
.pdf:lang(zh-TW) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK TC'; }
.pdf:lang(ko-KR) .mono { font-family: PT Mono, iawriter-mono, Nitti, Menlo, Courier, monospace, 'Twemoji', 'Noto Color Emoji', 'Noto Sans Mono CJK KR'; }
.highlight-default {
	color: rgba(55, 53, 47, 1);
}
.highlight-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.highlight-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.highlight-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.highlight-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.highlight-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.highlight-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.highlight-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.highlight-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.highlight-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.highlight-gray_background {
	background: rgba(241, 241, 239, 1);
}
.highlight-brown_background {
	background: rgba(244, 238, 238, 1);
}
.highlight-orange_background {
	background: rgba(251, 236, 221, 1);
}
.highlight-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.highlight-teal_background {
	background: rgba(237, 243, 236, 1);
}
.highlight-blue_background {
	background: rgba(231, 243, 248, 1);
}
.highlight-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.highlight-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.highlight-red_background {
	background: rgba(253, 235, 236, 1);
}
.block-color-default {
	color: inherit;
	fill: inherit;
}
.block-color-gray {
	color: rgba(120, 119, 116, 1);
	fill: rgba(120, 119, 116, 1);
}
.block-color-brown {
	color: rgba(159, 107, 83, 1);
	fill: rgba(159, 107, 83, 1);
}
.block-color-orange {
	color: rgba(217, 115, 13, 1);
	fill: rgba(217, 115, 13, 1);
}
.block-color-yellow {
	color: rgba(203, 145, 47, 1);
	fill: rgba(203, 145, 47, 1);
}
.block-color-teal {
	color: rgba(68, 131, 97, 1);
	fill: rgba(68, 131, 97, 1);
}
.block-color-blue {
	color: rgba(51, 126, 169, 1);
	fill: rgba(51, 126, 169, 1);
}
.block-color-purple {
	color: rgba(144, 101, 176, 1);
	fill: rgba(144, 101, 176, 1);
}
.block-color-pink {
	color: rgba(193, 76, 138, 1);
	fill: rgba(193, 76, 138, 1);
}
.block-color-red {
	color: rgba(212, 76, 71, 1);
	fill: rgba(212, 76, 71, 1);
}
.block-color-gray_background {
	background: rgba(241, 241, 239, 1);
}
.block-color-brown_background {
	background: rgba(244, 238, 238, 1);
}
.block-color-orange_background {
	background: rgba(251, 236, 221, 1);
}
.block-color-yellow_background {
	background: rgba(251, 243, 219, 1);
}
.block-color-teal_background {
	background: rgba(237, 243, 236, 1);
}
.block-color-blue_background {
	background: rgba(231, 243, 248, 1);
}
.block-color-purple_background {
	background: rgba(244, 240, 247, 0.8);
}
.block-color-pink_background {
	background: rgba(249, 238, 243, 0.8);
}
.block-color-red_background {
	background: rgba(253, 235, 236, 1);
}
.select-value-color-uiBlue { background-color: rgba(35, 131, 226, .07); }
.select-value-color-pink { background-color: rgba(245, 224, 233, 1); }
.select-value-color-purple { background-color: rgba(232, 222, 238, 1); }
.select-value-color-green { background-color: rgba(219, 237, 219, 1); }
.select-value-color-gray { background-color: rgba(227, 226, 224, 1); }
.select-value-color-translucentGray { background-color: rgba(255, 255, 255, 0.0375); }
.select-value-color-orange { background-color: rgba(250, 222, 201, 1); }
.select-value-color-brown { background-color: rgba(238, 224, 218, 1); }
.select-value-color-red { background-color: rgba(255, 226, 221, 1); }
.select-value-color-yellow { background-color: rgba(253, 236, 200, 1); }
.select-value-color-blue { background-color: rgba(211, 229, 239, 1); }
.select-value-color-pageGlass { background-color: undefined; }
.select-value-color-washGlass { background-color: undefined; }

.checkbox {
	display: inline-flex;
	vertical-align: text-bottom;
	width: 16;
	height: 16;
	background-size: 16px;
	margin-left: 2px;
	margin-right: 5px;
}

.checkbox-on {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20width%3D%2216%22%20height%3D%2216%22%20fill%3D%22%2358A9D7%22%2F%3E%0A%3Cpath%20d%3D%22M6.71429%2012.2852L14%204.9995L12.7143%203.71436L6.71429%209.71378L3.28571%206.2831L2%207.57092L6.71429%2012.2852Z%22%20fill%3D%22white%22%2F%3E%0A%3C%2Fsvg%3E");
}

.checkbox-off {
	background-image: url("data:image/svg+xml;charset=UTF-8,%3Csvg%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%0A%3Crect%20x%3D%220.75%22%20y%3D%220.75%22%20width%3D%2214.5%22%20height%3D%2214.5%22%20fill%3D%22white%22%20stroke%3D%22%2336352F%22%20stroke-width%3D%221.5%22%2F%3E%0A%3C%2Fsvg%3E");
}
	
</style></head><body><article id="a04e7e6e-a3ed-4ffb-8b70-a1f6def3b3f2" class="page sans"><header><div class="page-header-icon undefined"><span class="icon">ü§ñ</span></div><h1 class="page-title">An√°lisis metilaci√≥n O. maya</h1><p class="page-description"></p><table class="properties"><tbody><tr class="property-row property-row-multi_select"><th><span class="icon property-icon"><svg role="graphics-symbol" viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0" class="typesMultipleSelect"><path d="M1.91602 4.83789C2.44238 4.83789 2.87305 4.40723 2.87305 3.87402C2.87305 3.34766 2.44238 2.91699 1.91602 2.91699C1.38281 2.91699 0.952148 3.34766 0.952148 3.87402C0.952148 4.40723 1.38281 4.83789 1.91602 4.83789ZM5.1084 4.52344H14.3984C14.7607 4.52344 15.0479 4.23633 15.0479 3.87402C15.0479 3.51172 14.7607 3.22461 14.3984 3.22461H5.1084C4.74609 3.22461 4.45898 3.51172 4.45898 3.87402C4.45898 4.23633 4.74609 4.52344 5.1084 4.52344ZM1.91602 9.03516C2.44238 9.03516 2.87305 8.60449 2.87305 8.07129C2.87305 7.54492 2.44238 7.11426 1.91602 7.11426C1.38281 7.11426 0.952148 7.54492 0.952148 8.07129C0.952148 8.60449 1.38281 9.03516 1.91602 9.03516ZM5.1084 8.7207H14.3984C14.7607 8.7207 15.0479 8.43359 15.0479 8.07129C15.0479 7.70898 14.7607 7.42188 14.3984 7.42188H5.1084C4.74609 7.42188 4.45898 7.70898 4.45898 8.07129C4.45898 8.43359 4.74609 8.7207 5.1084 8.7207ZM1.91602 13.2324C2.44238 13.2324 2.87305 12.8018 2.87305 12.2686C2.87305 11.7422 2.44238 11.3115 1.91602 11.3115C1.38281 11.3115 0.952148 11.7422 0.952148 12.2686C0.952148 12.8018 1.38281 13.2324 1.91602 13.2324ZM5.1084 12.918H14.3984C14.7607 12.918 15.0479 12.6309 15.0479 12.2686C15.0479 11.9062 14.7607 11.6191 14.3984 11.6191H5.1084C4.74609 11.6191 4.45898 11.9062 4.45898 12.2686C4.45898 12.6309 4.74609 12.918 5.1084 12.918Z"></path></svg></span>Etiquetas</th><td></td></tr><tr class="property-row property-row-date"><th><span class="icon property-icon"><svg role="graphics-symbol" viewBox="0 0 16 16" style="width:14px;height:14px;display:block;fill:rgba(55, 53, 47, 0.45);flex-shrink:0" class="typesDate"><path d="M3.29688 14.4561H12.7031C14.1797 14.4561 14.9453 13.6904 14.9453 12.2344V3.91504C14.9453 2.45215 14.1797 1.69336 12.7031 1.69336H3.29688C1.82031 1.69336 1.05469 2.45215 1.05469 3.91504V12.2344C1.05469 13.6973 1.82031 14.4561 3.29688 14.4561ZM3.27637 13.1162C2.70898 13.1162 2.39453 12.8154 2.39453 12.2207V5.9043C2.39453 5.30273 2.70898 5.00879 3.27637 5.00879H12.71C13.2842 5.00879 13.6055 5.30273 13.6055 5.9043V12.2207C13.6055 12.8154 13.2842 13.1162 12.71 13.1162H3.27637ZM6.68066 7.38086H7.08398C7.33008 7.38086 7.41211 7.30566 7.41211 7.05957V6.66309C7.41211 6.41699 7.33008 6.3418 7.08398 6.3418H6.68066C6.44141 6.3418 6.35938 6.41699 6.35938 6.66309V7.05957C6.35938 7.30566 6.44141 7.38086 6.68066 7.38086ZM8.92285 7.38086H9.31934C9.56543 7.38086 9.64746 7.30566 9.64746 7.05957V6.66309C9.64746 6.41699 9.56543 6.3418 9.31934 6.3418H8.92285C8.67676 6.3418 8.59473 6.41699 8.59473 6.66309V7.05957C8.59473 7.30566 8.67676 7.38086 8.92285 7.38086ZM11.1582 7.38086H11.5547C11.8008 7.38086 11.8828 7.30566 11.8828 7.05957V6.66309C11.8828 6.41699 11.8008 6.3418 11.5547 6.3418H11.1582C10.9121 6.3418 10.8301 6.41699 10.8301 6.66309V7.05957C10.8301 7.30566 10.9121 7.38086 11.1582 7.38086ZM4.44531 9.58203H4.84863C5.09473 9.58203 5.17676 9.50684 5.17676 9.26074V8.86426C5.17676 8.61816 5.09473 8.54297 4.84863 8.54297H4.44531C4.20605 8.54297 4.12402 8.61816 4.12402 8.86426V9.26074C4.12402 9.50684 4.20605 9.58203 4.44531 9.58203ZM6.68066 9.58203H7.08398C7.33008 9.58203 7.41211 9.50684 7.41211 9.26074V8.86426C7.41211 8.61816 7.33008 8.54297 7.08398 8.54297H6.68066C6.44141 8.54297 6.35938 8.61816 6.35938 8.86426V9.26074C6.35938 9.50684 6.44141 9.58203 6.68066 9.58203ZM8.92285 9.58203H9.31934C9.56543 9.58203 9.64746 9.50684 9.64746 9.26074V8.86426C9.64746 8.61816 9.56543 8.54297 9.31934 8.54297H8.92285C8.67676 8.54297 8.59473 8.61816 8.59473 8.86426V9.26074C8.59473 9.50684 8.67676 9.58203 8.92285 9.58203ZM11.1582 9.58203H11.5547C11.8008 9.58203 11.8828 9.50684 11.8828 9.26074V8.86426C11.8828 8.61816 11.8008 8.54297 11.5547 8.54297H11.1582C10.9121 8.54297 10.8301 8.61816 10.8301 8.86426V9.26074C10.8301 9.50684 10.9121 9.58203 11.1582 9.58203ZM4.44531 11.7832H4.84863C5.09473 11.7832 5.17676 11.708 5.17676 11.4619V11.0654C5.17676 10.8193 5.09473 10.7441 4.84863 10.7441H4.44531C4.20605 10.7441 4.12402 10.8193 4.12402 11.0654V11.4619C4.12402 11.708 4.20605 11.7832 4.44531 11.7832ZM6.68066 11.7832H7.08398C7.33008 11.7832 7.41211 11.708 7.41211 11.4619V11.0654C7.41211 10.8193 7.33008 10.7441 7.08398 10.7441H6.68066C6.44141 10.7441 6.35938 10.8193 6.35938 11.0654V11.4619C6.35938 11.708 6.44141 11.7832 6.68066 11.7832ZM8.92285 11.7832H9.31934C9.56543 11.7832 9.64746 11.708 9.64746 11.4619V11.0654C9.64746 10.8193 9.56543 10.7441 9.31934 10.7441H8.92285C8.67676 10.7441 8.59473 10.8193 8.59473 11.0654V11.4619C8.59473 11.708 8.67676 11.7832 8.92285 11.7832Z"></path></svg></span>√öltima Edici√≥n</th><td><time>@17 de enero de 2024</time></td></tr></tbody></table></header><div class="page-body"><h2 id="a5dd260d-fd98-4f4e-8ecb-2cd20f6c5240" class="">Pruebas de an√°lisis de metilaci√≥n Ovario sin Genoma de Referencia (Approach: RefFreeDMA)</h2><p id="b5045419-b835-48dd-a827-a1cccf6ac885" class="">Se inici√≥ el an√°lisis bioinform√°tico de las lecturas de Ovario. Se realiz√≥ un subset de los datos para poder realizar pruebas de los an√°lisis. </p><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="58693fb3-93d6-44a8-aace-9a6a83fce4f8" class="code"><code class="language-JavaScript">#!/bin/bash
#SBATCH -p cicese
#SBATCH --job-name=subset
#SBATCH --output=subset-%j.log
#SBATCH --error=subset-%j.err
#SBATCH -N 1
#SBATCH --ntasks-per-node=8
#SBATCH -t 6-00:00:00


#head -n 40000 ./Ov2/H30R2_X5_L3_1.fq.gz &gt; ./Ov2/H30R2Subset_X5_L3_1.fq.gz
head -n 40000 ./Ov2/H30R2_X5_L3_2.fq.gz &gt; ./Ov2/H30R2Subset_X5_L3_2.fq.gz
head -n 40000 ./Ov2/H30R2_X5_L1_1.fq.gz &gt; ./Ov2/H30R2Subset_X5_L1_1.fq.gz
head -n 40000 ./Ov2/H30R2_X5_L1_2.fq.gz &gt; ./Ov2/H30R2Subset_X5_L1_2.fq.gz
head -n 40000 ./Ov3/H30R3_X5_L1_1.fq.gz &gt; ./Ov3/H30R3Subset_X5_L1_1.fq.gz
head -n 40000 ./Ov3/H30R3_X5_L1_2.fq.gz &gt; ./Ov3/H30R3Subset_X5_L1_2.fq.gz
head -n 40000 ./Ov3/H30R3_X5_L3_1.fq.gz &gt; ./Ov3/H30R3Subset_X5_L3_1.fq.gz
head -n 40000 ./Ov3/H30R3_X5_L3_2.fq.gz &gt; ./Ov3/H30R3Subset_X5_L3_2.fq.gz
head -n 40000 ./Ov4/H24R1_X5_L3_1.fq.gz &gt; ./Ov4/H24R1Subset_X5_L3_1.fq.gz
head -n 40000 ./Ov4/H24R1_X5_L3_2.fq.gz &gt; ./Ov4/H24R1Subset_X5_L3_2.fq.gz
head -n 40000 ./Ov4/H24R1_HNX5_L1_1.fq.gz &gt; ./Ov4/H24R1Subset_HNX5_L1_1.fq.gz
head -n 40000 ./Ov4/H24R1_HNX5_L1_2.fq.gz &gt; ./Ov4/H24R1Subset_HNX5_L1_2.fq.gz
head -n 40000 ./Ov4/H24R1_HVX5_L1_1.fq.gz &gt; ./Ov4/H24R1Subset_HVX5_L1_1.fq.gz
head -n 40000 ./Ov4/H24R1_HVX5_L1_2.fq.gz &gt; ./Ov4/H24R1Subset_HVX5_L1_2.fq.gz
head -n 40000 ./Ov5/H24R2_X7_L2_1.fq.gz &gt; ./Ov5/H24R2Subset_X7_L2_1.fq.g
head -n 40000 ./Ov5/H24R2_X7_L2_2.fq.gz &gt; ./Ov5/H24R2Subset_X7_L2_2.fq.gz
head -n 40000 ./Ov5/H24R2_X7_L1_1.fq.gz &gt; ./Ov5/H24R2Subset_X7_L1_1.fq.gz
head -n 40000 ./Ov5/H24R2_X7_L1_2.fq.gz &gt; ./Ov5/H24R2Subset_X7_L1_2.fq.gz 
head -n 40000 ./Ov5/H24R2_X5_L4_1.fq.gz &gt; ./Ov5/H24R2Subset_X5_L4_1.fq.gz
head -n 40000 ./Ov5/H24R2_X5_L4_2.fq.gz &gt; ./Ov5/H24R2Subset_X5_L4_2.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L1_1.fq.gz &gt; ./Ov6/H24R3Subset_X5_L1_1.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L1_2.fq.gz &gt; ./Ov6/H24R3Subset_X5_L1_2.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L3_1.fq.gz &gt; ./Ov6/H24R3Subset_X5_L3_1.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L3_2.fq.gz &gt; ./Ov6/H24R3Subset_X5_L3_2.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L2_1.fq.gz &gt; ./Ov6/H24R3Subset_X5_L2_1.fq.gz
head -n 40000 ./Ov6/H24R3_X5_L2_2.fq.gz &gt; ./Ov6/H24R3Subset_X5_L2_2.fq.gz

exit 0</code></pre><p id="0dbc5aba-a746-40ba-b1e5-01ca77fe64f6" class="">Actualizamos el scrip utilizando la funci√≥n ‚Äúfor‚Äù para automatizar los procesos de una manera m√°s eficiente. Creamos el siguiente script para automatizar el proceso anterior. </p><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="47e6c44e-d53f-4980-9642-930827b90f63" class="code"><code class="language-JavaScript">#!/bin/bash
# This script takes the first 10, 000 reads from fastq files, and saves it in a new fastq file. 

#my initials vars
DIR=&quot;$1&quot;
NUM_LINES=&quot;40000&quot;
DIR_FINAL=&quot;subset_reads&quot;

# Para crear el directorio de salida donde se van guardar los archivos 

if [ -d &quot;$DIR_FINAL&quot; ]; then
	echo &quot;removing last $DIR_FINAL directory&quot;
	rm -rf $DIR_FINAL
	#make dir
	echo &quot;creating new dir&quot;
	mkdir $DIR_FINAL 
fi

# my loop
for IN in $DIR/ *_1.fq.gz; do
	IN2=&quot;${IN/_1/_2}&quot;
	BASE_NAME_LIB1=&quot;${IN#raw_reads//}&quot;
	BASE_NAME_LIB2=&quot;${IN2#raw_reads//}&quot;
    BASE_NAME_L1=&quot;${BASE_NAME_LIB1%.fq.gz}&quot;
    BASE_NAME_L2=&quot;${BASE_NAME_LIB2%.fq.gz}&quot;
    echo &quot;BASE BASE_NAME_LIB1 IS: $BASE_NAME_LIB1&quot;
    echo &quot;BASE_NAME_LIB2 IS:  $BASE_NAME_LIB2&quot;
    echo processing &quot;$BASE_NAME_L1 and $BASE_NAME_L2 files&quot;
    zcat $IN | head -n $NUM_LINES &gt; ${BASE_NAME_L1}_subset.fq
    zcat $IN2 | head -n $NUM_LINES &gt; ${BASE_NAME_L2}_subset.fq 
   	gzip ${BASE_NAME_L1}_subset.fq
    gzip ${BASE_NAME_L2}_subset.fq
    mv ${BASE_NAME_L1}_subset.fq.gz $DIR_FINAL
    mv ${BASE_NAME_L2}_subset.fq.gz $DIR_FINAL
 done
</code></pre><p id="cb7cc192-60ba-4415-b042-8f2e51bda130" class="">
</p><p id="e816ca6d-c4cf-46ac-a448-3185db4d4d0f" class="">La realizaci√≥n del subset de datos se realiz√≥ en el cluster de Omica del cicese y posteriormente se descargaron los datos al disco local.</p><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="0d400da4-778f-4787-8290-9b6ddb698d75" class="code"><code class="language-JavaScript">scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov1/*Subset*.gz ./
scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov2/*Subset*.gz ./
scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov3/*Subset*.gz ./
scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov4/*Subset*.gz ./
scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov5/*Subset*.gz ./
scp sadot@158.97.9.9:/LUSTRE/bioinformatica_data/genomica_funcional/Sadot/RawData/Raw_Data/Ov6/*Subset*.gz ./</code></pre><p id="d432e14b-af52-4007-8527-a46ebdabdcee" class="">Tomando como base <a href="https://www.notion.so/RefFreeDMA-Jhonana-Klug-6d491a2ba94c4db1b661b65ea70cc173?pvs=21"><span class="icon">ü§ñ</span>RefFreeDMA - Jhonana Klug ‚úÖ</a> se inici√≥ la configuraci√≥n del script para poder utilizarlo con las muestras de Ovario </p><p id="968fc63d-b490-41d2-86a3-0ebb1d6cce12" class="">A continuaci√≥n se muestran las modificaciones realizas en el script .cfg</p><div id="c017a6cd-0eef-477c-be70-f84cf9859c00" class="column-list"><div id="5593c095-fec1-4b56-961e-ee2410ecae1a" style="width:50%" class="column"><h3 id="5790b9e3-7f91-48fe-b91f-f71cfd32a9e7" class="">Original</h3><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="2cba23d8-c5eb-46ea-827b-15ab46b82041" class="code"><code class="language-Bash">#-----------------------TOOLS_START----------------------------
tool_path=/home/sadotr/RRBS/RefFreeDMA/tools/
picard_path=$tool_path/picard-tools_1.118/
trim_galore_path=$tool_path/trim_galore_0.3.3/
cutadapt_path=$tool_path/cutadapt_1.8.3/lib/python2.7/site-packages/cutadapt/scripts/
bowtie2_path=$tool_path/bowtie_2.2.4/bin/
bsmap_path=$tool_path/bsmap_2.90/
samtools_path=$tool_path/samtools_1.2/bin/
bedtools_path=$tool_path/bedtools_2.26.0/bin/
bwa_path=$tool_path/bwa_0.7.8/bin/
bwameth_path=$tool_path/bwa-meth-0.10/
decon_reference=YOUR_DECON_PATH/decon_reference/bacterial_extracted_add
#-----------------------TOOLS_END----------------------------

#----------------------VARIABLES_START-------------------------
working_dir=/home/sadotr/RRBS/RefFreeDMA/RefFreeDMA_test
species=Hum
cross_genome_fa=-
sample_annotation=$working_dir/meta/sampleAnnotation.tsv
compCol=comp_gran_lympho
groupsCol=Cell_Type
parallel=FALSE
nonCpG=FALSE
#----------------------VARIABLES_END-------------------------

#----------------------DEFAULT_PARAMETERS_START-----------------------
decon=FALSE							#If set to TRUE all reads will be aligned to a microbial decoy genome and only reads that do not align will be used for building the deduced genome/further analysis.
restrictionSites=&quot;CGG&quot;						#Restriction sites (found at the beginning of RRBS reads) if multiple specify them like this(separated by pipe): &quot;CGG|CGA&quot; (MspI|Taq1)
wait_time=0.1							#Check every wait_time minutes weather process is finished (only relevant for parallel mode).
nProcesses=1							#Maximum number of allowed parallel processes for mapping and methylation calling.
nameSeparator=&quot;#&quot;						#Unique character(s) that separate flowdell ID from sample name (as indicated in the sample annotation sheet) in the bam-file name. 
										#If the bam-file name consists only of the sample name put &quot;&quot;
maxReadLen=51							#Actual maximum read letngth or if 3&#x27; croping is desired for, max length of read after cropping.
maxSamples=`ls $working_dir/unmapped_bam/*.bam|wc -l`	#Per default use all samples that are provided as .bam files in the unmapped_bam folder. Change this parameter to the number of samples you specified as selected in the sample annotation file if you only want to use a subset of samples for reference generation.
filtLim=4								#Only consider reads for the generation of the deduced genome that occure at least in 2 out of filtLim samples.
cLimit=0.05								#Minimum frequency of cytosines at a certain position in order to call a cytosine in the consensus sequence.
mapToSelf_filter=0.08					#Only accept hits as true matches where the mismatch rate is &lt;mapToSelf_filter
consensus_dist=0.05						#Max accepted mismatch rate between a read and its assigned consensus
crossMap_mismatchRate=0.2				#Allowed mismatch rate during crossmapping. The parameter is passed to bsmap
nTopDiffMeth=500						#Maximum number of top differentially methylated fragments to return as fasta sequences

#----------------------DEFAULT_PARAMETERS_END-----------------------</code></pre></div><div id="e7bd017e-ded2-4a14-b9c2-bce34d2ed8d3" style="width:50%" class="column"><h3 id="82dd442f-b6e1-42e1-8d08-6c243f141ea6" class="">Prueba Ovario</h3><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="67c2e95e-6456-460e-9c0c-a6288933ced4" class="code"><code class="language-Bash">#-----------------------TOOLS_START----------------------------
tool_path=/home/sadotr/RRBS/RefFreeDMA_PruebaOv/tools
picard_path=/home/sadotr/RRBS/RefFreeDMA_PruebaOv/tools/picard-tools_1.118/
trim_galore_path=$tool_path/trim_galore_0.3.3/
cutadapt_path=$tool_path/cutadapt_1.8.3/lib/python2.7/site-packages/cutadapt/scripts/
bowtie2_path=$tool_path/bowtie_2.2.4/bin/
bsmap_path=$tool_path/bsmap_2.90/
samtools_path=$tool_path/samtools_1.2/bin/
bedtools_path=$tool_path/bedtools_2.26.0/bin/
bwa_path=$tool_path/bwa_0.7.8/bin/
bwameth_path=$tool_path/bwa-meth-0.10/
decon_reference=YOUR_DECON_PATH/decon_reference/bacterial_extracted_add
#-----------------------TOOLS_END----------------------------
#----------------------VARIABLES_START-------------------------
working_dir=/home/sadotr/RRBS/RefFreeDMA_PruebaOv/RefFreeDMA_test
species=Om
cross_genome_fa=-
sample_annotation=$working_dir/meta/sampleAnnotation.tsv
compCol=Temperature
groupsCol=Temperature
parallel=FALSE
nonCpG=FALSE
#----------------------VARIABLES_END-------------------------
#----------------------DEFAULT_PARAMETERS_START-----------------------
decon=FALSE							#If set to TRUE all reads will be aligned to a microbial decoy genome and only reads that do not align will be used for building the deduced genome/further analysis.
restrictionSites=&quot;CGG&quot;						#Restriction sites (found at the beginning of RRBS reads) if multiple specify them like this(separated by pipe): &quot;CGG|CGA&quot; (MspI|Taq1)
wait_time=0.1							#Check every wait_time minutes weather process is finished (only relevant for parallel mode).
nProcesses=1							#Maximum number of allowed parallel processes for mapping and methylation calling.
nameSeparator=&quot;#&quot;						#Unique character(s) that separate flowdell ID from sample name (as indicated in the sample annotation sheet) in the bam-file name. 
										#If the bam-file name consists only of the sample name put &quot;&quot;
maxReadLen=51							#Actual maximum read letngth or if 3&#x27; croping is desired for, max length of read after cropping.
maxSamples=2 							#Per default use all samples that are provided as .bam files in the unmapped_bam folder. Change this parameter to the number of samples you specified as selected in the sample annotation file if you only want to use a subset of samples for reference generation.
filtLim=4								#Only consider reads for the generation of the deduced genome that occure at least in 2 out of filtLim samples.
cLimit=0.05								#Minimum frequency of cytosines at a certain position in order to call a cytosine in the consensus sequence.
mapToSelf_filter=0.08					#Only accept hits as true matches where the mismatch rate is &lt;mapToSelf_filter
consensus_dist=0.05						#Max accepted mismatch rate between a read and its assigned consensus
crossMap_mismatchRate=0.2				#Allowed mismatch rate during crossmapping. The parameter is passed to bsmap
nTopDiffMeth=500						#Maximum number of top differentially methylated fragments to return as fasta sequences
#----------------------DEFAULT_PARAMETERS_END-----------------------</code></pre></div></div><p id="bbea8875-44f7-4c06-9ede-f4c0933fd5cb" class="">A continuaci√≥n de corre el comando ejecutar el an√°lisis con las muestras prueba </p><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="6b3212ff-8bdf-4eed-bc51-d11fb062cd80" class="code"><code class="language-Bash">./RefFreeDMA.sh /home/sadotr/RRBS/RefFreeDMA_PruebaOv/RefFreeDMA_test/meta/RefFreeDMA_test.cfg</code></pre><h3 id="6b0102b4-dbfd-44fa-b5e3-3e4b001191c0" class="">Output en terminal </h3><script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/prism.min.js" integrity="sha512-7Z9J3l1+EYfeaPKcGXu3MS/7T+w19WtKQY/n+xzmw4hZhJ9tyYmcUS+4QqAlzhicE5LAfMQSF3iFTK9bQdTxXg==" crossorigin="anonymous" referrerPolicy="no-referrer"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css" integrity="sha512-tN7Ec6zAFaVSG3TpNAKtk4DOHNpSwKHxxrsiw4GHKESGPs5njn/0sMCUMl2svV4wo4BK/rCP7juYz+zx+l6oeQ==" crossorigin="anonymous" referrerPolicy="no-referrer"/><pre id="d867b5ce-dd60-4ff6-9aac-807ced2dccae" class="code"><code class="language-Bash">./RefFreeDMA.sh /home/sadotr/RRBS/RefFreeDMA_PruebaOv/RefFreeDMA_test/meta/RefFreeDMA_test.cfg

./RefFreeDMA.sh: line 16: /home/sadotr/RRBS/RefFreeDMA_PruebaOv/RefFreeDMA_test/meta/RefFreeDMA_test.cfg: No such file or directory
mkdir: cannot create directory ‚Äò/log‚Äô: Permission denied

Checking for required python libraries...
python Bio ... OK
python pysam ... OK
python bitarray ... OK
python guppy ... OK

Checking for required tools...
samtools ... OK
bedtools ... OK
trim_galore ... OK
cutadapt ... OK
bowtie2 ... OK
bsmap ... OK
samToFastq.jar ...  FAIL

Please provide missing python libraries and tools.
If you are using the external software bundle, did you set the path in the configuration file?</code></pre></div></article><span class="sans" style="font-size:14px;padding-top:2em"></span></body></html>